.gitignore

```diff
@@ -17,3 +17,5 @@ dist/
 docs/_build
 venv/
 htmlcov/
+.pyre
+.pyre_configuration
```

Comment:
what is pyre?

Reply:
A static type checker (http://pyre-check.org/), but I switched to Mypy (https://mypy.readthedocs.io/) now in #134 as it seems to be more complete and easier to configure